model/godleyTableWindow.h

```diff
@@ -67,7 +68,7 @@ namespace minsky
       godleyIcon(godleyIcon), idx(idx) {}
   };
 
-  class GodleyTableWindow: public ecolab::CairoSurface, public ButtonWidgetEnums
+  class GodleyTableWindow: public ecolab::CairoSurface, public ButtonWidgetEnums, public GodleyAssetClass
```

Comment:
The main reason for inheriting from a class enum is to gain access to enumerator labels. I don't think the small simplification of the type declaration on line 173 warrants this.

Reply:
If I don't include it, I get the following error:

`model/godleyTableWindow.h:175:56: error: ‘AssetClass’ was not declared in this scope
     std::deque<std::pair<GodleyTable::Data,std::vector<AssetClass>>> history;
                                                        ^~~~~~~~~~
model/godleyTableWindow.h:175:56: note: suggested alternative: ‘Tk_SetClass’
     std::deque<std::pair<GodleyTable::Data,std::vector<AssetClass>>> history;
                                                        ^~~~~~~~~~
`

This is after having included the headers of godleyTable and assetClass in godleyTableWindow.h. Very weird...

Reply:
Yes - that is why you have to refer to the type as GodleyAssetClass::AssetClass.